source/wp-content/themes/wporg-news-2021/sass/components/_categories.scss

```diff
@@ -2,125 +2,266 @@
  * Styles for the default category archives, and its variations.
  */
 
-body.category-community {
-
-	@extend %local-header-off-white;
+body.category .site-content-container {
+	padding-left: var(--wp--custom--layout--content-meta-size);
+	margin: var(--wp--custom--alignment--edge-spacing);
+
+	// Some categories include a decorative drop-cap-like character.
```

Comment:
This is a cool approach, kudos 👍🏻 

You may already be planning to, but do you mind breaking this off into a separate PR for #75? This PR is already huge and hard to review w/ just the refactor stuff. It seems like there's a couple other non-refactor things as well.

If it's not too much trouble, it'd be really helpful to break the refactor stuff into multiple PRs too.

Reply:
Thanks.

I think the changes in this PR are so large and comprehensive that it might not make sense to break it down into smaller PRs — each PR would be dependent on the others; There are some changes that could happen independently, but the PR as a whole works much better as a cohesive change.